### DISCUSSION TITLE
Can't fabricate items

### DISCUSSION THREAD

**Discussion by lelun** [FakeFishGames/Barotrauma#14377] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

unable to use fabricators or vending machne in multiplayer. so when i decontstruct a item in multiplayer i klick the button wait for it to deconstruct and a message apears in the event log saying i have deconstructed somthing. when i try to do the same with my constructor lets say i want to craft any item but in this case a canister shell i go to the constroctur and place the ingredients i had just deconstructed in there and then pressed the construct botton. issue starts here, it looks like i klick the buttion but it wont react and no event shows that the constructor got activated. same happens on vending machines i klick the botton, it looks like it klicks but nothing happens. it works in single player but it wont work when i self host a server. i have tried to verify steam files. uninstalled and re installed the game, manualy removed the game from my pc + appdata folder. but nothing seams to work. i have also tried to change game version. 

bugs happen both in the sommer hotfix 2 and unstable version.

![20240731172443_1](https://github.com/user-attachments/assets/7267b150-34ba-42cf-b9c9-e1e4719d4ee4)
![20240731172404_1](https://github.com/user-attachments/assets/101a41e6-5208-48fc-82aa-b73886e7a036)
![20240731172437_1](https://github.com/user-attachments/assets/c6bb25de-569d-4d4d-b668-d553ed358b17)

### Reproduction steps

start game. take any item and deconstruct it. then try to build anything with the resorces you have gotten from the materials. or use a vending machine. it wont do any actions when klicked.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by lelun** [comment]
also tried the mac compatebility mode just to check unable to klick the button there either in self hosted mp. i  assume that there is somthing wrong with my keybindings or somthing but i am unable to determen what it is

**Comment by lelun** [comment]
as a controll i just removed the entire game again and made sure steam did not back anything up issue is still there

**Comment by Regalis11** [comment]
Thank you for the report! I unfortunately could not find a way to reproduce the issue: fabricating seems to work with no issues at my end, so I suspect this requires some more specific circumstances to occur.

![image](https://github.com/user-attachments/assets/5efbf1fb-14ba-460f-b599-d6c619f9c765)

When you click the button, does anything at all happen? Does the button react to the click (go down and play the click sound), does the text on it change, do you see the green "progress bar" on the output slot...?

**Reply by lelun** [reply]
it dues not make the klick sound and but it reacts by making the drepressed animation. text dues not change its like parts of the program recognizes that there is a klick but its not rellayed that it should start the process. are there any way for me to generate a log that shows everything happening in game?

i have tried to unbind the left mouse botton and change the key used for that interaction. that button seams to be seperate to the other interaction button. i dont know if thats a intended feature or not. are there any other files then appdata and the steam folder its located in that i could try to remove?

issue only happens on self hosted multiplayer. wile it works in the campaign.
